packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -3498,7 +3498,7 @@ describe("IModelTransformerHub", () => {
         master: {
           manualUpdate(db) {
             const notDeleted = db.elements.deleteDefinitionElements([
-              spatialViewDef.id,
+              spatialViewDef!.id,
```

Comment:
Not sure what happened, but when updating packages some how this broke, and "!" was needed. This test is skipped, so I don't know about its importance, but I am concerned I updated something I should not have